synapse/tests/test_tools_migrate020.py

```diff
@@ -791,7 +791,7 @@ class MigrationTest(s_t_utils.SynTest):
     async def test_migr_dirn(self):
         with self.getRegrDir('cortexes', REGR_VER) as src:
             with self.getTestDir() as dest:
-                locallyrs = os.listdir(os.path.join(src, 'layers'))
+                os.listdir(os.path.join(src, 'layers'))
```

Comment:
Is this doing anything?

Reply:
I believe it will raise if there doesn't exist a directory with that name.